src/storages/key_value_store.js

```diff
@@ -292,6 +292,7 @@ export class KeyValueStore {
      * @param {object} [options] All `forEachKey()` parameters are passed
      *   via an options object with the following keys:
      * @param {string} [options.exclusiveStartKey] All keys up to this one (including) are skipped from the result.
+     * @param {number} [index=0]
```

Comment:
This was omitted on purpose I think. Users should not change this as its used for the recursion.

Reply:
Just because you omit a parameter from jsdoc doesn't meet users won't be able to change it... If we go for typescript, this will be implicit anyway. I kinda hate to see those quirks as IDE does highlight it with warnings :]

If we really want to forbid usage of the param, we should use a private method with such parameter, and remove it from the public one.